hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaRolloutManagement.java

```diff
@@ -624,6 +624,7 @@ private void createScheduledAction(final Collection<Target> targets, final Distr
             action.setStatus(Status.SCHEDULED);
             action.setRollout(rollout);
             action.setRolloutGroup(rolloutGroup);
+            action.setTriggeredBy(rollout.getCreatedBy());
```

Comment:
todo: verfiy when rollout is in state waiting approval, who should be the trigggerer here?

Reply:
I still believe that the triggerer here would be the creator. If I want to do something but have to wait for approval, I'm still the triggerer of the action, not the approver. Without the creator, there would be no action

Reply:
@bogdan-bondar what do you think?